include/galsim/SBInterpolatedImage.h

```diff
@@ -248,20 +146,75 @@ namespace galsim {
         void calculateMaxK(double max_maxk=0.) const;
 
         ConstImageView<double> getImage() const;
-        boost::shared_ptr<Interpolant> getXInterp() const;
-        boost::shared_ptr<Interpolant> getKInterp() const;
 
     protected:
 
         class SBInterpolatedImageImpl;
 
-        // Regular SBProfile pimpl constructor so as to be available to derived classes
-        SBInterpolatedImage(SBProfileImpl* pimpl) : SBProfile(pimpl) {}
+        // Wow, three classes here... I'm not even sure how this works,
+        // but the tests seem to pass...
```

Comment:
Ha!  Yeah, this was my way of calling attention to this line to see if I got it right!  Let's see if @rmjarvis has any comment here...